ansible/roles/ocp4-workload-integreatly/tasks/post_workload.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Implement your Post Workload deployment tasks here
+# --------------------------------------------------
+
+
+
+
+# Leave these as the last tasks in the playbook
+# ---------------------------------------------
+
+# For deployment onto a dedicated cluster (as part of the
+# cluster deployment) set workload_shared_deployment to False
+# This is the default so it does not have to be set explicitely
+- name: pre_workload tasks complete
+  debug:
+    msg: "Post-Workload tasks completed successfully."
+  when:
+  - not silent|bool
+  - not workload_shared_deployment|d(False)
+
+# For RHPDS deployment (onto a shared cluster) set
+# workload_shared_deployment to True
+# (in the deploy script or AgnosticV configuration)
+- name: pre_workload tasks complete
```

Comment:
Same here ^^

Reply:
Oh. Yeah. I'll fix that in the example. The important part (the message) is correct. It's just the name of the task. Thanks for spotting. Duh.